src/components/POLine/Vendor/VendorForm.test.js

```diff
@@ -38,4 +42,29 @@ describe('VendorForm', () => {
     expect(screen.getByText('ui-orders.vendor.accountNumber')).toBeInTheDocument();
     expect(screen.getByText('ui-orders.vendor.instructions')).toBeInTheDocument();
   });
+
+  it('should render active account numbers', () => {
+    const accounts = [
+      {
+        name: 'name',
+        accountNo: '1',
+        accountStatus: 'Active',
+      },
+      {
+        name: 'name2',
+        accountNo: '2',
+        accountStatus: 'Inactive',
+      },
+      {
+        name: 'name3',
+        accountNo: '3',
+        accountStatus: 'Active',
+      },
+    ];
+
+    renderVendorForm({ accounts });
+
+    expect(screen.getByText(`${accounts[0].name} (${accounts[0].accountNo})`)).toBeInTheDocument();
+    expect(screen.getByText(`${accounts[0].name} (${accounts[0].accountNo})`)).toBeInTheDocument();
```

Comment:
These lines are equal, either remove one of them, or fix if there is a typo in the array index